SDM.h

```diff
@@ -264,10 +362,15 @@ class SDM {
     uint16_t getMsTimeout();                                                    //  get current value of RESPONSE_TIMEOUT (ms)
 
   private:
+
+    bool validChecksum(const uint8_t* data, size_t messageLength) const;
+
+    void modbusWrite(uint8_t* data, size_t messageLength);
+
 #if defined ( USE_HARDWARESERIAL )
     HardwareSerial& sdmSer;
 #else
-    SoftwareSerial& sdmSer;
+    ESPeasySerial& sdmSer;
```

Comment:
to keep the library universal, I think softwareserial should be left

Reply:
Sure, I have forgotten to replace that one.